coord/src/module.c

```diff
@@ -1931,7 +1931,15 @@ int initSearchCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
       tableSize = 16384;
   }
 
-  MRCluster *cl = MR_NewCluster(initialTopology, sf, 2);
+  size_t num_connections_per_shard;
+  if (clusterConfig.connPerShard) {
```

Comment:
why not covered?

Reply:
because currently there are no tests for coordinator configurations, and no test is setting this value